commit 6c8a44d404f4732f7758ea547875f1396573921a (refs/changes/64/864/2)
Author: Patrick Mooney <pmooney@pfmooney.com>
Date:   2016-11-09T10:54:44-06:00 (2 years, 11 months ago)
    
    PUBAPI-1346 allow ufds_master credentials to differ from local ufds

diff --git a/sapi_manifests/cloudapi/template b/sapi_manifests/cloudapi/template
index 34fd725..fa36b0e 100644
--- a/sapi_manifests/cloudapi/template
+++ b/sapi_manifests/cloudapi/template
@@ -28,7 +28,12 @@
     "ufds_master": {
         "url": "ldaps://{{{ufds_remote_ip}}}",
         "bindDN": "{{{ufds_ldap_root_dn}}}",
-        "bindPassword": "{{{ufds_ldap_root_pw}}}",
+        {{^ufds_remote_ldap_root_pw}}
+        "bindPassword": "{{{ufds_ldap_root_pw}}}"
+        {{/ufds_remote_ldap_root_pw}}
+        {{#ufds_remote_ldap_root_pw}}
+        "bindPassword": "{{{ufds_remote_ldap_root_pw}}}"
+        {{/ufds_remote_ldap_root_pw}}
         "cache": {
             "size": 5000,
             "age": 60
